test/server/cards/02_SHD/units/TarffulKashyyykChieftain.spec.ts

```diff
@@ -0,0 +1,57 @@
+describe('Tarfful, Kashyyyk Chieftain', function() {
+    integration(function(contextRef) {
+        describe('Tarfful\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['tarfful#kashyyyk-chieftain', 'liberated-slaves', 'isb-agent'],
+                    },
+                    player2: {
+                        groundArena: ['wampa', 'specforce-soldier', 'wroshyr-tree-tender', 'atst'],
+                        spaceArena: ['cartel-spacer'],
+                    }
+                });
+            });
+
+            it('when a wookie unit is damaged that unit deals that much damage to an enemey ground unit', function () {
+                const { context } = contextRef;
+
+                // Scenario 1: a friendly wookie unit attacks
+                context.player1.clickCard(context.liberatedSlaves);
+                context.player1.clickCard(context.wroshyrTreeTender);
+
+                expect(context.player1).toBeAbleToSelectExactly([context.wampa, context.specforceSoldier, context.wroshyrTreeTender, context.atst]);
+                context.player1.clickCard(context.specforceSoldier);
+
+                expect(context.specforceSoldier).toBeInZone('discard', context.player2);
```

Comment:
We should do at least one check with a unit that isn't defeated by the ability damage so we can confirm the amount is right